client/app/components/user/session/user-session.service.js

```diff
@@ -370,9 +370,6 @@ class SessionService {
                         state: "app.account.useraccount.emails"
                     },
                     (this.constants.target === "EU") && {
```

Comment:
you forgot to remove the conditional

Reply:
Did not understand how I worked, I will remove it